nixos/modules/services/backup/borgbackup.nix

```diff
@@ -292,6 +300,35 @@ in {
             '';
           };
 
+          randomizedDelaySec = mkOption {
+              default = "0";
+              type = types.str;
+              example = "45min";
+              description = ''
+                  Add a randomized delay before each backup run.
+                  The delay will be chosen between zero and this value.
+                  This value must be a time span in the format specified by
+                  <citerefentry><refentrytitle>systemd.time</refentrytitle>
+                  <manvolnum>7</manvolnum></citerefentry>
+                  '';
+          };
+
+          persistent = mkOption {
+              default = true;
+              type = types.bool;
+              example = false;
+              description = ''
+                  Takes a boolean argument. If true, the time when the service
+                  unit was last triggered is stored on disk. When the timer is
+                  activated, the service unit is triggered immediately if it
+                  would have been triggered at least once during the time when
+                  the timer was inactive. Such triggering is nonetheless
+                  subject to the delay imposed by RandomizedDelaySec=. This is
+                  useful to catch up on missed runs of the service when the
+                  system was powered down.
+                  '';
+          };
```

Comment:
Please follow the formatting of the rest of file (indentation with 2 spaces, close `''` indented at the same level as where it was opened).